src/CacheTool/Adapter/FastCGI.php

```diff
@@ -21,6 +21,11 @@ class FastCGI extends AbstractAdapter
      */
     protected $client;
 
+    protected $possibleSocketFiles = [
+        '/var/run/php5-fpm.sock',
+        '/var/run/php/php7.0-fpm.sock'
```

Comment:
`/var/run/php-fpm.sock` would be nice to add too, as a generic name.. or maybe glob for `/var/run/php*.sock` / `/var/run/php/*.sock`  as a last resort?